cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -192,15 +192,16 @@ func TestAddFlags(t *testing.T) {
 				{Group: "apiregistration.k8s.io", Resource: "apiservices"},
 				{Group: "apiextensions.k8s.io", Resource: "customresourcedefinitions"},
 			},
-			NodeEvictionRate:                  0.2,
-			SecondaryNodeEvictionRate:         0.05,
-			LargeClusterSizeThreshold:         100,
-			UnhealthyZoneThreshold:            0.6,
-			DisableAttachDetachReconcilerSync: true,
-			ReconcilerSyncLoopPeriod:          metav1.Duration{Duration: 30 * time.Second},
-			Controllers:                       []string{"foo", "bar"},
-			EnableTaintManager:                false,
-			UseServiceAccountCredentials:      true,
+			NodeEvictionRate:                      0.2,
+			SecondaryNodeEvictionRate:             0.05,
+			LargeClusterSizeThreshold:             100,
+			UnhealthyZoneThreshold:                0.6,
+			DisableAttachDetachReconcilerSync:     true,
+			ReconcilerSyncLoopPeriod:              metav1.Duration{Duration: 30 * time.Second},
+			Controllers:                           []string{"foo", "bar"},
+			EnableTaintManager:                    false,
+			HorizontalPodAutoscalerUseRESTClients: true,
```

Comment:
Nit pick: you're setting this to the default, so probably it's not needed.

Reply:
It's not default value for this struct, I added this to fix failing unit test.